content/extraProblems.json

```diff
@@ -472,6 +472,110 @@
         "usacoId": "1142"
       }
     },
+    {
+      "uniqueId": "usaco-1156",
+      "name": "Air Cownditioning",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1156",
+      "source": "Bronze",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["Greedy"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1156"
+      }
+    },
+    {
+      "uniqueId": "usaco-1159",
+      "name": "Connecting Two Barns",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1159",
+      "source": "Silver",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Connected Components"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1159"
+      }
+    },
+    {
+      "uniqueId": "usaco-1160",
```

Comment:
Maybe add this to module. It wouldn't hurt to add a "hard" prefix sums problem to the module (consider there is only one).

Reply:
From DM: I don't think convoluted intervals teaches you anything new about prefix sums. The point was just that you should use M being small somehow